src/load.jl

```diff
@@ -0,0 +1,217 @@
+using Ghost
+using Ghost: Tape, Input, mkcall, Variable, V
+
+
+struct ONNXCtx
+    name2var::Dict{String, Variable}
+    backends::Vector{Symbol}
+    exec::Bool
+end
+
+ONNXCtx(backends; exec=true) = ONNXCtx(Dict(), backends, exec)
+ONNXCtx(;exec=true) = ONNXCtx(Dict(), [:ONNX], exec)
+
+# TODO: implement rebind_context!()
+
+"""
+    getindex(tape::Tape{ONNXCtx}, onnx_name::String)
+
+Get operation on the tape using the name in ONNX graph
+"""
+Base.getindex(tape::Tape{ONNXCtx}, onnx_name::String) =
+    tape[tape.c.name2var[onnx_name]]
+
+###############################################################################
+#                               Operations                                    #
+###############################################################################
+
+
+mrev(x) = x
+mrev(x::AbstractVector) = reverse(x)
+prev(x) = x
+prev(x::AbstractVector) = reshape(permutedims(reverse(reshape(x, length(x) ÷ 2,:);dims=1)),:)
+
+
+# mrev = maybe reverse. prev = rearrange padding, e.g. (1,2,1,2) => (2,2,1,1) or (1,2,3,1,2,3) => (3,3,2,2,1,1)
+_akpsd(params) = get(params, :activation, identity), mrev(get(params, :kernel_shape, 1)), prev(get(params, :pads, 0)), mrev(get(params, :strides, 1)), mrev(get(params, :dilations, 1))
+akpsd(params) = a2t.(_akpsd(params))
+a2t(x) = x
+a2t(a::AbstractArray) = Tuple(a)
```

Comment:
Call this something like `conv_params`?

Reply:
Same here, should be covered by `onnx2julia()`/`julia2onnx()`.